README.md

```diff
@@ -15,6 +23,9 @@ Or directly from our seed node:
 
     cargo install radicle-cli --force --locked --git https://seed.alt-clients.radicle.xyz/radicle-cli.git
 
+> **Note**
+> It's recommended to install pre-built binaries since development builds may be incompatible with the network.
+
```

Comment:
:+1: maybe 'release' binaries instead? would linking to the release page make sense here?